tech/database/redis/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Redis
+subsection: redis
+section: tech-database
+description: In-memory data structure store, used as a database, cache and message broker
+---
+
+# Redis
+
+Redis is a free and open source in memory data structure store. It is widely used as a database, in memory cache or a message broker.
+
+# Quick start
+
+Redis installation on fedora
+
+```sh
+$ sudo dnf install redis     # Install redis cli and server
+$ sudo systemctl start redis # Initialize redis server
+```
+
+To start redis on boot, run
+
+```sh
+sudo systemctl enable redis
+```
+
+To test, run
+
+```sh
+$ redis-cli ping # output will be pong
```

Comment:
Consider adding the expected output under this command and with a description along the lines of `Result of running the test looks like this`.